src/create-entity-api.ts

```diff
@@ -28,12 +28,12 @@ import {
  * @param {ReturnType<typeof createApiCreator> | undefined} [options.baseApiCreator] - The APIs creator generated by `createApiCreator` that shares configuration for new APIs.
  * @param {BaseQueryFunction | undefined} [options.baseQuery] - The base query function.
  * @param {ClassConstructor<TEntity>} options.entityConstructor - The entity model class constructor.
- * @param {ClassConstructor<TSearchRequest> | typeof PaginationRequest} [options.entitySearchRequestConstructor] - The search request class constructor. Defaults to PaginationRequest.
- * @param {ClassConstructor<TEntityRequest> | typeof EntityRequest} [options.entityGetRequestConstructor] - The get request class constructor. Defaults to EntityRequest.
- * @param {ClassConstructor<TSearchResponse> | typeof PaginationResponse} [options.entitySearchResponseConstructor] - The search response class constructor. Defaults to PaginationResponse.
+ * @param {ClassConstructor<TSearchRequest> | typeof PaginationRequest} [options.entitySearchRequestConstructor=PaginationRequest] - The search request class constructor.
+ * @param {ClassConstructor<TEntityRequest> | typeof EntityRequest} [options.entityGetRequestConstructor=EntityRequest] - The get request class constructor.
+ * @param {ClassConstructor<TSearchResponse> | typeof PaginationResponse} [options.entitySearchResponseConstructor=PaginationResponse] - The search response class constructor.
  * @param {Array<EntityEndpointName>} [options.omitEndpoints] - The array of endpoints to omit. Defaults to never.
```

Comment:
`never` is a type. Let's remove `Defaults to never` and add `=[]` inside the square brackets

Reply:
Shouldn't we do `omitEndpoints = []` on 46 line then? 
Instead of `omitEndpoints || []` on 290th line.

Reply:
Yes, let's do it

Reply:
Changed as discussed